frontend/src/metabase/reference/databases/TableDetail.jsx

```diff
@@ -39,6 +40,7 @@ const interestingQuestions = table => {
         dbId: table.db_id,
         tableId: table.id,
         getCount: true,
+        metadata,
```

Comment:
All `getQuestionUrl` calls need `metadata` now

Reply:
how did it work before? 🤯

Reply:
Previously the query was constructed as JSON with simple manipulations and hardcoded clauses (see diff in `frontend/src/metabase/reference/utils.js`). Metadata was only used for fields.

Now as everything needs to go through MLv2 query, metadata is always needed.